addon/i3dio/ui/object.py

```diff
@@ -872,6 +893,42 @@ def draw(self, context):
         row.prop(obj.i3d_mapping, 'mapping_name')
 
 
+@persistent
+def check_lod_distance(dummy):
+    for obj in bpy.data.objects:
+        if obj.type == 'EMPTY':
+            if hasattr(obj, 'i3d_attributes') and 'lod_distance' in obj['i3d_attributes']:
+                current_lod = obj['i3d_attributes']['lod_distance']
+                # Skip if already converted
+                if not isinstance(current_lod, str):
+                    continue
+
+                # Check for placeholder or empty strings
+                if current_lod in ('', 'Enter your LOD Distances if needed.'):
+                    print(f"LOD distance for {obj.name} is not set.")
+                    continue
+
+                try:
+                    # Convert from string to list of floats
+                    lod_distance_str = current_lod.split()
+                    new_lod_distance = [float(x) for x in lod_distance_str]
+                    new_lod_distance = (new_lod_distance + [0.0] * 4)[:4]
+
+                    # first value should always be 0.0
+                    new_lod_distance[0] = 0.0
+
+                    # Each subsequent value cannot be smaller than the previous one
+                    for i in range(1, len(new_lod_distance)):
+                        if new_lod_distance[i] < new_lod_distance[i - 1]:
+                            new_lod_distance[i] = new_lod_distance[i - 1]
+
+                    obj.i3d_attributes.lod_distance = new_lod_distance
+                    print(f"Converted LOD distance for {obj.name}: {new_lod_distance}")
+
+                except (ValueError, AttributeError) as e:
+                    print(f"Error converting LOD distance for {obj.name}: {e}")

```

Comment:
I would get rid of the prints as we don't print any information about conversions in the other places where we convert and it's just noise in the log.

```suggestion
                except (ValueError, AttributeError) as e:
                    pass
```